Added: alt attribute to few images (#19366)
mdn/content: files/en-us/glossary/state_machine/index.md

_Figure 1: Deterministic Finite State Machine_

> ![This is the image of a Deterministic Finite State Machine where the machine transitions from state 1 to state 2 for input X and it transitions from state 1 to state 3 for input Y](statemachine1.png)

Reviewer: ```suggestion
![The machine transitions from state 1 to state 2 for input X and from state 1 to state 3 for input Y](statemachine1.png)
```

In _Figure 1_, the state begins in State 1; the state changes to State 2 given input 'X', or to State 3 given input 'Y'.

_Figure 2: Non-Deterministic Finite State Machine_

> ![This is the image of a Non-Deterministic Finite State Machine where the machine remains in state 1 for input X and it transitions from state 1 to state 2 for the same input X](statemachine2.png)

Reviewer: ```suggestion
![The machine may remain in state 1, transitioning to itself, or may transition from state 1 to state 2 for input X](statemachine2.png)
```
